tests/e2e/config/bootstrap.js

```diff
@@ -122,6 +122,38 @@ function observeConsoleLogging() {
 	} );
 }
 
+/**
+ * Runs Axe tests when the block editor is found on the current page.
+ *
+ * @return {?Promise} Promise resolving once Axe texts are finished.
+ */
+async function runAxeTestsForBlockEditor() {
```

Comment:
Can we rename this to `runAxeTestsForStoriesEditor` and use a different selector to detect it?

Reply:
The block editor and stories editor will have different configurations though. The `exclude` config already has a stories-specific config in it as well.

Also, eventually we should eventually add these tests for the plugin settings screen as well, where we're going to have yet another set of configuration options.

Having three separate sets would make maintenance easier.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3294/commits/78161f9d42a87c9ca636467daaa96705ee4540aa

Reply:
The selector in `runAxeTestsForStoriesEditor` is still the same though.

What I meant was having separate functions with separate configurations:

* `runAxeTestsForBlockEditor` with the current `.block-editor` selector
* `runAxeTestsForStoriesEditor` with a `#amp-story-editor` selector
* Eventually: `runAxeTestsForSettingsScreen` with a `#amp-settings` selector
* Eventually: `runAxeTestsForValidationScreen` with a `.post-type-amp_validated_url` selector

Reply:
> The selector in `runAxeTestsForStoriesEditor` is still the same though.
> 
> What I meant was having separate functions with separate configurations:
> 
> * `runAxeTestsForBlockEditor` with the current `.block-editor` selector
> * `runAxeTestsForStoriesEditor` with a `#amp-story-editor` selector
> * Eventually: `runAxeTestsForSettingsScreen` with a `#amp-settings` selector
> * Eventually: `runAxeTestsForValidationScreen` with a `.post-type-amp_validated_url` selector

What are you asking to change here? Just change the selector, like this `runAxeTestsForStoriesEditor` with a `#amp-story-editor` selector

Reply:
Yes. **And** have a **second function**, `runAxeTestsForBlockEditor`, with a different selector that only targets the block editor but **not** the stories editor. That function would then not `exclude` `'.amp-story-page-attachment-close-button'` as it's **irrelevant** in the standalone block editor.

If it's unclear I can also change this myself.

Reply:
And of course, let me know if you think it doesn't make sense. Then we can just do the `#amp-story-editor` selector change for now.